compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/utils.kt

```diff
@@ -129,21 +132,41 @@ fun MemberDescriptor.toKSModifiers(): Set<Modifier> {
     if (this.isExternal) {
         modifiers.add(Modifier.EXTERNAL)
     }
+    val isStatic = this.hasJvmStaticAnnotation() || (this.containingDeclaration as? ClassDescriptor)?.let { containingClass ->
```

Comment:
This looks to be partially redundant with the `modifiers` getter in `KSDeclarationImpl` above. But I am confused as the test case is not showing 2 `JAVA_STATIC` for these static members, can you address my confusion here?

Reply:
KSDeclaration is the one that is used for kotlin sources whereas this method is used in the Descriptor implementation.
Am I missing something?

Reply:
Oh you are right, somehow I was reading this is in `KtModifierListOwner.toKSModifiers()`